docs/cvl/expr.md

```diff
@@ -365,6 +365,15 @@ while verifying the rule, and will provide a separate verification report for
 each checked method.  Rules that use this feature are referred to as
 {term}`parametric rule`s.
 
+Another possible way to have the Prover consider options for a function is by
+prefixing the function name with an `address` typed variable. In this case the
```

Comment:
Maybe this is slightly simpler wording? Prefixing the function name with `address` type variable reads a bit overly complicated? 

```suggestion
Another possible way to have the Prover consider options for a function is by
using an `address` typed variable and "calling" the function on it, e.g. `address a; a.foo(...);`. In this case the
```